crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -723,6 +723,20 @@ impl OlmMachine {
         self.group_session_manager.share_group_session(room_id, users, encryption_settings).await
     }
 
+    /// Receive a decrypted verification events.
```

Comment:
Method name says 'unencrypted', this says 'decrypted' but with a note that it does not need to be called for encrypted events. Shouldn't this line say 'unencrypted' too?

Reply:
Right, they should, fixed by https://github.com/matrix-org/matrix-rust-sdk/pull/616/commits/5535ab63a12ab2629aae412ba7a5d52c3d4502ff and https://github.com/matrix-org/matrix-rust-sdk/pull/616/commits/2f4382fc070758976e02d5775cdc6ddc1c592419.